src/Validator/Constraints/PhoneNumber.php

```diff
@@ -54,12 +54,6 @@ class PhoneNumber extends Constraint
      */
     public function __construct($format = null, $type = null, string $defaultRegion = null, string $regionPath = null, string $message = null, array $groups = null, $payload = null, array $options = [])
     {
-        if (\is_array($format)) {
-            $options = array_merge($format, $options);
-        } elseif (null !== $format) {
-            $options['value'] = $format;
-        }
```

Comment:
This looks indeed somehow wrong. Maybe it's done on purpose? cc @jderusse

Reply:
It kinda looks to me to be some BC code to some REALLY old SF versions (2.x maybe).
I've gone through the code trying to find any use for it but could not (tests passing without it also suggests it wasn't used anywhere).
But yeah some extra confirmation might be good